static/app/views/organizationStats/teamInsights/teamStability.tsx

```diff
@@ -180,35 +193,88 @@ class TeamStability extends AsyncComponent<Props, State> {
     const groupedProjects = groupByTrend(sortedProjects);
 
     return (
-      <StyledPanelTable
-        isEmpty={projects.length === 0}
-        headers={[
-          t('Project'),
-          <RightAligned key="last">{tct('Last [period]', {period})}</RightAligned>,
-          <RightAligned key="curr">{t('Last 7 Days')}</RightAligned>,
-          <RightAligned key="diff">{t('Difference')}</RightAligned>,
-        ]}
+      <SessionsRequest
+        api={api}
+        project={projects.map(({id}) => Number(id))}
+        organization={organization}
+        interval="1d"
+        groupBy={['session.status', 'project']}
+        field={[SessionField.SESSIONS]}
+        statsPeriod={period}
       >
-        {groupedProjects.map(({project}) => (
-          <Fragment key={project.id}>
-            <ProjectBadgeContainer>
-              <ProjectBadge avatarSize={18} project={project} />
-            </ProjectBadgeContainer>
-
-            <ScoreWrapper>{this.renderScore(project.id, 'period')}</ScoreWrapper>
-            <ScoreWrapper>{this.renderScore(project.id, 'week')}</ScoreWrapper>
-            <ScoreWrapper>{this.renderTrend(project.id)}</ScoreWrapper>
-          </Fragment>
-        ))}
-      </StyledPanelTable>
+        {({response, loading}) => (
+          <StyledPanelTable
+            isEmpty={projects.length === 0}
+            headers={[
+              t('Project'),
+              <RightAligned key="last">{tct('Last [period]', {period})}</RightAligned>,
+              <RightAligned key="avg">{tct('[period] Avg', {period})}</RightAligned>,
+              <RightAligned key="curr">{t('Last 7 Days')}</RightAligned>,
+              <RightAligned key="diff">{t('Difference')}</RightAligned>,
+            ]}
+          >
+            {groupedProjects.map(({project}) => {
+              const sumSessions =
+                response &&
+                getSeriesSum(
+                  response.groups.filter(
+                    group => group.by.project === Number(project.id)
+                  ),
+                  SessionField.SESSIONS,
+                  response.intervals
+                );
+
+              return (
+                <Fragment key={project.id}>
+                  <ProjectBadgeContainer>
+                    <ProjectBadge avatarSize={18} project={project} />
+                  </ProjectBadgeContainer>
+
+                  <div>
+                    {response && !loading && sumSessions && (
+                      <MiniBarChart
+                        isGroupedByDate
+                        showTimeInTooltip
+                        series={[
+                          {
+                            seriesName: t('Crash Free Sessions'),
+                            data: getCountSeries(
+                              SessionField.SESSIONS,
+                              response.groups.find(
+                                g =>
+                                  g.by.project === Number(project.id) &&
```

Comment:
something here feels very deeply nested, might be nice to make another function in this file that handles rendering the chart

Reply:
Moved series to its own function.